tools/setup_helpers/cuda.py

```diff
@@ -72,7 +72,7 @@ def find_cuda_version(cuda_home):
     if IS_LINUX or IS_DARWIN:
         CUDA_HOME = os.getenv('CUDA_HOME', LINUX_HOME)
     else:
-        CUDA_HOME = os.getenv('CUDA_PATH', None).replace('\\', '/')
+        CUDA_HOME = os.getenv('CUDA_HOME', None).replace('\\', '/')
```

Comment:
Noo! Sorry if you misunderstood me! I meant to leave `CUDA_PATH` if this is what standard CUDA installation sets by default!